packages/cli-core/package.json

```diff
@@ -7,17 +7,17 @@
   },
   "dependencies": {
     "chalk": "^2.4.2",
-    "execa": "^1.0.0",
-    "fs-extra": "^7.0.1",
-    "globby": "^9.0.0",
-    "inquirer": "^6.2.2",
+    "execa": "^2.0.4",
+    "fs-extra": "^8.1.0",
+    "globby": "^10.0.1",
+    "inquirer": "^7.0.0",
     "is-ci": "^2.0.0",
-    "log-symbols": "^2.2.0",
-    "ora": "^3.1.0",
-    "read-pkg-up": "^6.0.0",
-    "slash": "^2.0.0",
+    "log-symbols": "^3.0.0",
+    "ora": "^4.0.2",
+    "read-pkg-up": "^7.0.0",
```

Comment:
this did create a breaking change https://github.com/4Catalyzer/javascript/pull/402/files#diff-f2c8515dd0711c9a0d09b3e5446fbe78R35